src/wp-includes/l10n.php

```diff
@@ -57,7 +57,7 @@ function get_locale() {
 			$ms_locale = get_site_option( 'WPLANG' );
 		} else {
 			$ms_locale = get_option( 'WPLANG' );
-			if ( false === $ms_locale ) {
+			if ( ! $ms_locale ) {
```

Comment:
This doesn't seem right. If `WPLANG` returns an empty string it means that the site has set the locale to `en_US` and should not fall back to the site option.

Reply:
Hmmm you're right. Added an extra phpunit test for that case so we don't miss it and will keep working on this.

Though it's a puzzle not that easy to crack. Ideally `en_US` should not be an empty string but properly defined like all other languages... :thinking: